sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/analysis/FunctionRegistry.scala

```diff
@@ -660,8 +662,13 @@ object FunctionRegistry {
     }
     val clazz = scala.reflect.classTag[Cast].runtimeClass
     val usage = "_FUNC_(expr) - Casts the value `expr` to the target data type `_FUNC_`."
-    val expressionInfo =
-      new ExpressionInfo(clazz.getCanonicalName, null, name, usage, "", "", "", "", "", "")
+    val examples = s"""
+      |    Examples:
+      |      > SELECT _FUNC_($exampleIn);
+      |       $exampleOut
```

Comment:
We need this example? IMO the `usage` statement looks enough for documents.  Mixing up code and examples in L554-565 looks a bit weird.